ceph-api-nightly/config/definitions/ceph-api-nightly.yml

```diff
@@ -2,6 +2,7 @@
     name: ceph-api-nightly
     ceph_branch:
       - master
+      - quincy
       - pacific
       - octopus
       - nautilus
```

Comment:
Can we remove nautilus now?

Reply:
We discussed the same thing on our standup last day and our decision was to run nautilus too because its only for the nightlies. @epuertat What do you think?

Reply:
Yup, given it's just once a day @djgalloway if that doesn't take much capacity from other stuff, can we keep nautilus for a while? Even if it's technically EoL, [there have been multiple contributions](https://github.com/ceph/ceph/commits/nautilus) after the last release (Jun 30, 2021).

Reply:
Sure